root/admin/wikidoc/WikiDocIndex.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *      /*  ((n   *'.
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {defaults} from 'lodash';
+
+import Layout from '../../layout';
+import Table from '../../components/Table';
+import {withCatalystContext} from '../../context';
+
+import type {WikiDocT} from './types';
+
+type PropsT = {
+  +$c: CatalystContextT,
+  +pages: $ReadOnlyArray<WikiDocT>,
+  +updates_required: boolean,
+  +wiki_server: string,
+  +wiki_unreachable: boolean,
+};
+
+const WikiDocTable = withCatalystContext(({
+  $c,
+  pages,
+  updates_required,
+  wiki_server,
+}: PropsT) => {
+  const columns = React.useMemo(
+    () => {
+      const nameColumn = {
+        Cell: ({cell: {value}}) => (
+          <a href={'/doc/' + encodeURIComponent(value)}>{value}</a>
+        ),
+        Header: N_l('Page name'),
+        accessor: 'id',
+        cellProps: {className: 'title'},
+        headerProps: {className: 'c'},
+        id: 'name',
+      };
+      const transcludedVersionColumn = {
+        Header: N_l('Transcluded version'),
+        accessor: 'version',
+        cellProps: defaults(
+          {className: 'c transcluded-version'},
+          (updates_required && $c.user?.is_wiki_transcluder
+            ? {style: {textAlign: 'right'}}
+            : {}
```

Comment:
My guess is it allows to check if updates are required without having to scroll the whole list.

Reply:
I dunno, I'm one of the main people using this and I honestly never realized there was any other setup (since pretty much always there is some update required anyway by the time I see it). If what we want is a simple indicator, I'd maybe make the wikiVersionColumn header red instead if there's something to do there?

Reply:
Yes, or just a message :100: complete on top of that list. It should be changed carefully because wiki version cannot always be determined. And there are more small bugs and potential improvements to this page, that would be worth a separate PR. (I’m also using this form frequently.)